base/dict.jl

```diff
@@ -392,6 +393,22 @@ function setindex!(h::Dict{K,V}, v0, key::K) where V where K
     return h
 end
 
+function setindex!(h::Dict{K,Any}, v, key::K) where K
```

Comment:
why not `setindex!(h::Dict{K,Any}, @nospecialize v, key::K)`?

Reply:
I have options, so I use thehm